jest/private/jest_test.bzl

```diff
@@ -98,6 +98,9 @@ def _impl(ctx):
         # useful if the user configures jest-junit themselves.
         fixed_env["JEST_JUNIT_OUTPUT_FILE"] = "$$XML_OUTPUT_FILE"
 
+    if ctx.attr.quiet_snapshot_updates:
+        fixed_env["JS_BINARY__SILENT_ON_SUCCESS"] = "1"
```

Comment:
Is using `JS_BINARY__SILENT_ON_SUCCESS` like `js_run_binary` ok or is that considered "private"?

Reply:
That one I think we can consider public as it is very useful and not likely to every change.

Could even add it to the docs of js_binary